### DISCUSSION TITLE
etcd filer store - "number of entries in a folder ~10GB"

### DISCUSSION THREAD

**Discussion by lightelfua** [seaweedfs/seaweedfs#2176] [General]
Hi, i have a question about etcd filer store. I see in wiki - "number of entries in a folder ~10GB".
If i'm have directory format /files/photo/1/123/12345/1234567/ and file size summary in this directory must be smaller 10gb? Right?

**Comment by chrislusf** [comment]
it should be fine.

I updated the wiki which is off. 10GB is the upper range of total size of etcd filer store itself.

**Reply by lightelfua** [reply]
https://etcd.io/docs/v3.5/dev-guide/limit/

"Storage size limit
The default storage size limit is 2GB, configurable with --quota-backend-bytes flag. 8GB is a suggested maximum size for normal environments and etcd warns at startup if the configured value exceeds it."

Ok, we need to use other filer store. Thank you.
